nilearn/surface/tests/test_surface.py

```diff
@@ -150,6 +169,33 @@ def test_load_surf_mesh_list():
     del mesh
 
 
+def test_load_surf_mesh_img_to_data():
+    if not LooseVersion(nb.__version__) >= LooseVersion('2.1.0'):
+        raise SkipTest
```

Comment:
If I understand correctly _load_surf_mesh_img_to_data will work even with nibabel 2.0.2, so why skip the test? is it because building the image with GiftiDataArray and GiftiImage is different in 2.0.2?

Reply:
because of argument ```intent```. It is not there for nibabel 2.0.2 ? Am i correct ?

Reply:
and if you set it after the construction?
```
coord_array = gifti.GiftiDataArray(data=mesh[0])
coord_array.intent = nb.nifti1.intent_codes['NIFTI_INTENT_POINTSET']
# ...
```
seems to work in 2.0.2

Reply:
Wow! master of nibabel